pkgs/by-name/ss/ssh-openpgp-auth/daemon.nix

```diff
@@ -0,0 +1,8 @@
+{ callPackage }:
+
+callPackage ./generic.nix {
```

Comment:
I too wanted that, but it won't pass CI, here's the error you'd get:

```
Running nixpkgs-check-by-name..
pkgs/by-name/ss/sshd-openpgp-auth: File package.nix at line 3 contains the path expression "../ssh-openpgp-auth/generic.nix" which may point outside the directory of that package.
Validation failed, see above errors
Cleaning up.. Done
```

Reply:
oh huh.  i guess only packages in the legacy category hierarchy are allowed to intermingle.  maybe just add a comment at the top of `daemon.nix` explaining why it is called via `all-packages.nix`, otherwise it might seem like the file is unused.

Reply:
> oh huh. i guess only packages in the legacy category hierarchy are allowed to intermingle. maybe just add a comment at the top of `daemon.nix` explaining why it is called via `all-packages.nix`, otherwise it might seem like the file is unused.

Sure, sounds good to me.